configure

```diff
@@ -26,7 +27,7 @@ function is_windows() {
 }
 
 function is_ppc64le() {
-  [[ "${uname -m}" == "ppc64le" ]]
+  [[ "${CPU}" == "ppc64le" ]]
```

Comment:
Instead of defining another global variable, which has the risk of clashing at some point in the future, it's safer to do:
[[ "$(uname -m)" == "ppc64le" ]]